development/scripts/remove-rn-startup-performance-tracking.js

```
@@ -0,0 +1,173 @@
+
+    // Delete ready file if exists
+    if (fs.existsSync(readyFilePath)) {
+      fs.unlinkSync(readyFilePath);
+      stats.readyFilesDeleted++;
+      console.log(`✓ Deleted ready file: ${path.relative(rootDir, readyFilePath)}`);
+    }
+
+    let newContent = content;
+
+    // Remove import statement
+    const importPattern = new RegExp(`import '\\.\\/${fileNameWithoutExt}\\.perfReady';\\n`, 'g');
```

ESLint

[error] Replace ``import·'\\.\\/${fileNameWithoutExt}\\.perfReady';\\n`,·'g'` with `⏎······`import·'\\.\\/${fileNameWithoutExt}\\.perfReady';\\n`,⏎······'g',⏎····`

(prettier/prettier)